package Operation_maths;

import static org.junit.Assert.*;

import org.junit.Before;
import org.junit.Test;

public class DivisionTest {

	Division div;
	
	@Before
	public void setUp() throws Exception {
		div = new Division();
	}

	@Test
	public void Test1() {
		assertEquals(1, div.exec(42, 42));
	}
	
	@Test
	public void Test2() {
		assertEquals(2, div.exec(8,4));
	}
	
	@Test(expected=ArithmeticException.class)
	public void Test0() {
		div.exec(23, 0);
	}

}
